easybuild/easyblocks/o/openblas.py

```diff
@@ -44,3 +61,42 @@ def sanity_check_step(self):
             'dirs': [],
         }
         super(EB_OpenBLAS, self).sanity_check_step(custom_paths=custom_paths)
+    
+    def _parse_optarch( self, cfg_optarch):
+        """ Plucks the march out of the optarch settings. See
+        https://easybuild.readthedocs.io/en/latest/Controlling_compiler_optimization_flags.html """
+
+        pieces = cfg_optarch.split('=')
+        if len(pieces) > 1:
+            if pieces[0] == 'march':
+                return pieces[1]
+            elif pieces[2] == '-march':
+                return pieces[3]
+        else:
+            return cfg_optarch
+
+    def _get_openblas_targets(self, targetfile):
+        """ Parse the openblas target file and generate a list of targets """
+        targets = []
+
+        with open( targetfile ) as f:
+            lines = f.readlines()
```

Comment:
Use `lines = read_file(targetfile).splitlines()`, and have a backup plan in case the specified file doesn't exist?

Reply:
Yup, fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/1946/commits/d4da6d55bc8cdfe8534cfd4455adf07adcc5d972